src/css/components/_explore.scss

```diff
@@ -0,0 +1,66 @@
+.explore {
+  display: flex;
+  flex-direction: column;
+  min-height: 800px;
+
+  &__title {
+    font-size: 32px;
+    color: $gray3;
+  }
+
+  &__components {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    margin: 24px auto;
+  }
+
+  &__card {
+    display: flex;
+    max-width: 200px;
+    height: 300px;
+    align-items: flex-end;
+    border-radius: 8px;
+    margin-right: 24px;
+  }
+  
+  &__card--mariana {
+    background-image: url("/assets/explore/roteiro-mariana.png");
```

Comment:
@PinhoOn the images will come from the API, so we must not set them via css. Let's use `<img>` instead 👍